### ISSUE TITLE
Glossary escapes HTML

### ISSUE THREAD

**Issue by johnlindquist** [GitbookIO/gitbook#609]
I'd like to be able to use windows/mac icons in the glossary, but the Glossary seems to escape all HTML tags:

![image](https://cloud.githubusercontent.com/assets/36073/6192098/10d2b3b8-b36c-11e4-943f-441596f0a0cb.png)

**Comment by AaronO** [comment]
Are you using GitBook.com ? Which GitBook release are you using the `2.0.0-alpha` or `1.5.0` ?

**Comment by johnlindquist** [comment]
2.0.0-alpha7

Building locally w/ gitbook serve

**Comment by AaronO** [comment]
@johnlindquist Ok thanks,

This can be fixed pretty easily (a few different ways), the easiest of which being to change `glossary.html`'s (https://github.com/GitbookIO/gitbook/blob/version/2.0/theme/templates/website/glossary.html#L9) from

```
<p>{{ item.description }}</p>
```

to

```
{% autoescape false %}<p>{{ item.description }}</p>{% endautoescape %}
```

This should be pretty straightforward and easy to fix, nonetheless I'll let @SamyPesse make the final decision (and ship a new alpha tonight or tomorrow).

@SamyPesse While researching this, there's a small issue in the `2.0` branch, https://github.com/GitbookIO/gitbook/blob/version/2.0/lib/utils/page.js#L218, doesn't escape the glossary item's description when injecting it as a `title` attribute on the `a` link it creates.

@johnlindquist :+1: On the use case, it makes sense to have more elaborate glossaries whenever possible :)

**Comment by AaronO** [comment]
@johnlindquist If you're building locally, you can monkey-patch that in the meantime, @SamyPesse is out of the office right now. So it may take a few hours for him to ship the update.

**P.S**: Congrats on the great work with https://egghead.io !

**Comment by johnlindquist** [comment]
Thanks @AaronO . Congrats on gitbook, it's awesome. I'm writing a WebStorm book with it right now :)

**Comment by SamyPesse** [comment]
Ok, I'll look into that but I can't promise to ship a new release before Tuesday.

**Comment by johnlindquist** [comment]
No rush on my end :)

**Comment by SamyPesse** [comment]
Fixed in `2.0.0-alpha.8`

**Comment by johnlindquist** [comment]
Confirmed working in 2.0.0-alpha.8
